tapdb/sqlite.go

```diff
@@ -140,7 +144,41 @@ func NewSqliteStore(cfg *SqliteConfig) (*SqliteStore, error) {
 	// Now that the database is open, populate the database with our set of
 	// schemas based on our embedded in-memory file system.
 	if !cfg.SkipMigrations {
-		if err := s.ExecuteMigrations(TargetLatest); err != nil {
+		backupAndMigrate := func(mig *migrate.Migrate) error {
```

Comment:
Should we keep the mechanism with the `MigrationTarget`? So we can do a backup while also being able to target a specific version? Would be handy for unit tests.

Reply:
Hard for a skip backup flag to reach that point.